embed.fnc

```diff
@@ -4457,6 +4457,7 @@ S	|void	|give_perl_locale_control				\
 S	|parse_LC_ALL_string_return|parse_LC_ALL_string 		\
```

Comment:
From the commit message:
```
 ./Configure -DNO_LOCALE_TIME
```
Should this be:
```
 ./Configure -Accflags=-DNO_LOCALE_TIME
```
?

Reply:
Yes; fixed